diff --git a/entity/src/models/record.rs b/entity/src/models/record.rs
index 6c0150d..ebd314e 100644
--- a/entity/src/models/record.rs
+++ b/entity/src/models/record.rs
@@ -8,7 +8,9 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   pub name: String,
   pub zone_id: Uuid,
diff --git a/entity/src/models/zone.rs b/entity/src/models/zone.rs
index bc00e87..04625d6 100644
--- a/entity/src/models/zone.rs
+++ b/entity/src/models/zone.rs
@@ -8,7 +8,9 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   pub name: String,
   pub verified: bool,
