diff --git a/lib/entity/src/models/record.rs b/lib/entity/src/models/record.rs
--- a/lib/entity/src/models/record.rs
+++ b/lib/entity/src/models/record.rs
@@ -8,7 +8,9 @@
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   pub name: String,
   pub zone_id: Uuid,
diff --git a/lib/entity/src/models/zone.rs b/lib/entity/src/models/zone.rs
--- a/lib/entity/src/models/zone.rs
+++ b/lib/entity/src/models/zone.rs
@@ -8,7 +8,9 @@
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   pub name: String,
   pub verified: bool,
diff --git a/lib/entity/src/models/user.rs b/lib/entity/src/models/user.rs
--- a/lib/entity/src/models/user.rs
+++ b/lib/entity/src/models/user.rs
@@ -8,7 +8,9 @@
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   #[sea_orm(unique)]
   pub name: String,
diff --git a/lib/entity/src/models/password.rs b/lib/entity/src/models/password.rs
--- a/lib/entity/src/models/password.rs
+++ b/lib/entity/src/models/password.rs
@@ -8,7 +8,9 @@
 pub struct Model {
   #[sea_orm(primary_key, auto_increment = false)]
   pub id: Uuid,
+  #[serde(with = "time::serde::iso8601")]
   pub created: TimeDateTimeWithTimeZone,
+  #[serde(with = "time::serde::iso8601")]
   pub updated: TimeDateTimeWithTimeZone,
   pub hash: String,
 }
